### DISCUSSION TITLE
Add either delete or open in file explorer to the context menu

### DISCUSSION THREAD

**Discussion by isle9** [jeffvli/feishin#82] [Ideas]
I'd like to be able to delete songs directly or somewhat directly from feishin.
Delete should ideally delete to the OS trash folder if implemented.

If that's not possible, hopefully we can open the folder where the file is located.

**Comment by jeffvli** [comment]
I think this is technically possible.

Navidrome's API returns the path to the file but it's based off the music path set in the Navidrome config.
For someone running Navidrome on docker or a machine that's separate from their client, the path would be different.

I would have to add a customizable transformation for the path, as well as toggles to enable/disable the feature.

I'll look into this for a future relase.
